src/routes.tsx

```diff
@@ -155,29 +155,30 @@ export const reactPageRoutes: ReactPageRoute[] = [
     render: ({ match }: RouteComponentProps<any>) => {
       const getProps = () => {
         match.params
```

Comment:
`match.params` on this line can be removed now right? :)

Reply:
Yes and probably it wasn't even needed before either - like it doesn't do anything? 🤔 😄 Anyway, well spotted, will remove